docs/src/Landing.js

```diff
@@ -0,0 +1,140 @@
+//  Copyright (c) 2018-present, GM Cruise LLC
+//
+//  This source code is licensed under the Apache License, Version 2.0,
+//  found in the LICENSE file in the root directory of this source tree.
+//  You may not use this file except in compliance with the License.
+
+import React from 'react';
+import Logo from './Logo';
+import styled from 'styled-components';
+import GithubLogo from './GithubLogo';
+import CruiseLogo from './CruiseLogo';
+import { Link } from 'react-router-dom';
+import Demo from './Demo';
+
+const OuterContainer = styled.div`
+  padding: 50px;
+  background-image: url(wv.png);
+  background-position: center top;
+  background-size: 1600px auto;
+  background-repeat: no-repeat;
+  min-height: 100vh;
+  @media only screen and (max-width: 940px) {
+    background-position: left top;
+  }
+`;
+
+const InnerContainer = styled.div`
+  max-width: 1200px;
+  margin: 0 auto;
+`;
+
+const Header = styled.div`
+  max-width: 560px;
+  height: 700px;
+  padding: 30px 0;
+
+  h1,
+  h2 {
+    font-weight: normal;
+    margin: 24px 0;
+  }
+
+  h1 {
+    font-size: 38px;
+  }
+  h2 {
+    font-size: 20px;
+  }
+
+  @media only screen and (max-width: 940px) {
+    h1 {
+      font-size: 32px;
+    }
+  }
+`;
+
+const Flex = styled.div`
+  display: flex;
+  align-items: center;
+  flex-wrap: wrap;
+
+  a {
+    color: #edcc28;
+    border: 1px solid #edcc28;
+    padding: 4px 10px;
+    &:hover,
+    &:visited {
+      color: #edcc28;
+    }
+  }
+
+  @media only screen and (max-width: 940px) {
+    a {
+      width: 180px;
+      margin-bottom: 20px;
+    }
+  }
+`;
+
+const GithubLink = styled.a`
+  display: inline-flex;
+  align-items: center;
+  margin-right: 26px;
+
+  svg {
+    margin-right: 8px;
+  }
+`;
+
+const Footer = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin: 60px 0 0;
+`;
+
+const DocsFooterLink = styled(Link)`
+  display: block;
+  color: #edcc28 !important;
+  font-size: 18px;
+`;
+
+const Landing = () => (
+  <OuterContainer>
+    <InnerContainer>
```

Comment:
Add a redirect here to redirect to the root url:
```
   <Route exact path="/">
      {({ match }) => !match && <Redirect to="/" />}
    </Route>
```

Reply:
I added the redirect to the end of the switch in `App.js`:
```
<Switch>
    <Route path="/docs" component={Docs} />
    <Route exact path="/" component={Landing} />
    <Redirect to="/" />
</Switch>
```